packages/riipen-ui/src/components/Tabs.jsx

```diff
@@ -8,6 +8,11 @@ class Tabs extends React.Component {
   static displayName = "Tabs";
 
   static propTypes = {
+    /**
+     * The breakpoint to display the mobile tab styling. Use "none" for no styling.
+     */
+    breakpoint: PropTypes.oneOf(["sm", "md", "lg", "xl", "none"]),
```

Comment:
Basically a pass-through to the Tab components.  Makes it easier to set when using UI kit